docs/getting_started.md

```diff
@@ -877,8 +878,6 @@ Generate an ECDSA key:
     mbedtls_psa_crypto_free();
 ```
 
-### More about the Mbed Crypto library
-
-More information on [Mbed Crypto](https://github.com/ARMmbed/mbed-crypto/).
+### More about the Mbed Crypto API
```

Comment:
The Mbed Crypto library doesn't just implement the PSA Crypto API, it also implements the crypto part of the Mbed TLS API. The title of this section doesn't match the content anymore.